_data/governance.yml

```diff
@@ -0,0 +1,40 @@
+cards:
+  - title: "Scala Improvement Process"
+    description: "The SIP is the primary mechanism for evolving the Scala language."
+    expandText: "Process & People"
+    links:
+      - name: SIP Committee Members
+        url: https://docs.scala-lang.org/sips/process-specification.html#the-sip-committee
+      - name: SIP Process Specification
+        url: https://docs.scala-lang.org/sips/process-specification.html
+  - title: "Scala 3 Maintenance"
+    description: "Managing the Scala 3 compiler, standard library and documentation."
+    expandText: "Process & People"
+    links:
+      - name: Core Maintainers
+        url: https://github.com/lampepfl/dotty/blob/main/MAINTENANCE.md#compiler
+      - name: Maintenance Procedures
+        url: https://github.com/lampepfl/dotty/blob/main/MAINTENANCE.md
+  - title: "Tooling"
+    description: "Managing build tools, linters, IDEs, and other tools."
+    expandText: "Process & People"
+    links:
+      - text: "Process and People are TBD, please stay tuned! However you can read below about the Scala Tooling Summit:"
+      - name: "Scala Tooling Summit of March 2023"
+        url: https://www.scala-lang.org/blog/2023/04/11/march-2023-scala-tooling-summit.html
+  - title: "Moderation"
+    description: "Learn about how is the Scala community moderated."
+    expandText: "Process & People"
+    links:
+      - name: "Moderation Team"
+        url: https://www.scala-lang.org/conduct/#contact
+      - name: Code of Conduct
+        url: https://www.scala-lang.org/conduct/
+      - name: "Inclusive Language Guide"
+        url: https://docs.scala-lang.org/contribute/inclusive-language-guide.html
+      - name: "Communication Channels"
+        url: https://docs.scala-lang.org/community/
+  - title: "Update Frequency"
+    description: "This page is updated every 6 months by the Scala Center."
+    dontExpand: true
+    links: []
```

Comment:
The `Update Frequency` is kind of a meta-information about the governance. I suggest to move it to some introduction paragraph between the title `Governance` and the cards.

```
# Governance

This page is updated every 6 months by the Scala Center

## The Scala Improvement Process

...

## The Scala 3 Maintenance

....

## The Tooling

...

## The Moderation

...
```